client/my-sites/stats/all-time/index.jsx

```diff
@@ -36,16 +36,25 @@ module.exports = React.createClass( {
 
 	render: function() {
 		var bestDay = null,
+			statTabs = [ 'posts', 'views', 'visitors', 'best' ],
+			classSets = {},
 			infoIcon = this.state.showInfo ? 'info' : 'info-outline',
 			valueClass,
 			bestViews,
 			classes;
 
+		statTabs.forEach( function( tabName ) {
+			var tabClassOptions = {};
+			tabClassOptions[ 'module-tab' ] = true;
+			tabClassOptions[ 'is-' + tabName ] = true;
+			classSets[ tabName ] = classNames( tabClassOptions );
+		} );
```

Comment:
What's the purpose of the `tabClassOptions` variable? Could this not be done directly?

```js
classSets[ tabName ] = classNames( {
	'module-tab': true,
	`is-${ tabName }`: true
} );
```

Also at this point it would be worth considering an immutable construct over `forEach`, but that's not too important here.